src/datasource.ts

```diff
@@ -466,14 +467,17 @@ export default class CogniteDatasource {
       return [{value: "ERROR: Query can only use '='"}];
     }
 
-    let url = this.url + `/cogniteapi/${this.project}/events/search?`;
-    for (let param of queryOptions.filters) {
-      url += param.property + '=' + param.value + "&";
-    }
-
     // use maxStartTime and minEndTime so that we include events that are partially in range
+    const queryParams = Object.assign({
```

Comment:
Check out the spread operator over `Object.assign` for readability,
```js
const firstObject = { a: 2 };
const myObject = {
  a: 1,
  c: 2,
  ...a
};
```
will yield
```js
{ a: 2, c: 2 }
```